releasetasks/test/__init__.py

```diff
@@ -12,3 +13,10 @@ def read_file(path):
 OTHER_PUB_KEY = read_file(os.path.join(os.path.dirname(__file__),
                                        "other_rsa.pub"))
 DUMMY_PUBLIC_KEY = os.path.join(os.path.dirname(__file__), "public.key")
+
+
+def create_test_args(non_standard_arguments):
```

Comment:
<3 all of this. so much cleaner!

I wonder if 

1) `test_graph_parameters.yml` should be `default_graph_parameters.yml`?

2) is `test_graph_parameters.yml` too Firefox specific? Maybe it should live in `releasetasks/test/firefox/default_graph_parameters.yml`? Then we could decorate `create_test_args` under releasetasks/test/firefox/__init__.py with a `create_firefox_test_args`?

depends how generic and flexible we want this for other products. At any rate, 👍